drv.go

```diff
@@ -5,30 +5,30 @@
 
 // Package godror is a database/sql/driver for Oracle DB.
 //
-// The connection string for the sql.Open("godror", connString) call can be
+// The connection string for the sql.Open("godror", dataSourceName) call can be
 // the simple
 //   login/password@sid [AS SYSDBA|AS SYSOPER]
 //
 // type (with sid being the sexp returned by tnsping),
 // or in the form of
-//   ora://login:password@sid/? \
-//     sysdba=0& \
-//     sysoper=0& \
-//     poolMinSessions=1& \
-//     poolMaxSessions=1000& \
-//     poolIncrement=1& \
-//     connectionClass=& \
-//     standaloneConnection=0& \
-//     enableEvents=0& \
-//     heterogeneousPool=0& \
-//     prelim=0& \
-//     poolWaitTimeout=5m& \
-//     poolSessionMaxLifetime=1h& \
-//     poolSessionTimeout=30s& \
-//     timezone=Local& \
+//   host:port/service_name? \n
```

Comment:
Isn't this supposed to be "connectString=host:port/service_name" and the trailing "?" removed? Perhaps with a footnote as you said earlier regarding the old format that is still supported?

Reply:
You're right, this is the old oracle://... format.